mobile/lib/features/trade/position_list_item.dart

```diff
@@ -70,6 +71,10 @@ class _PositionListItemState extends State<PositionListItem> {
 
     TextStyle dataRowStyle = const TextStyle(fontSize: 14);
 
+    Amount? unrealizedPnl = notNullPosition.unrealizedPnl;
+    double pnlPercent =
+        ((unrealizedPnl?.sats ?? Amount.zero().sats) / notNullPosition.collateral.sats) * 100.0;
```

Comment:
🔧 we should probably round here to 2 numbers after the comma.

Reply:
Thanks for pointing this out. 

It's done when the pnlPercent is being used with `toStringAsFixed(2)`.  (I initially used the wrong function, but now it's correct).